crates/shared/src/http_solver_api.rs

```diff
@@ -14,6 +14,7 @@ pub trait HttpSolverApi: Send + Sync {
         &self,
         model: &model::BatchAuctionModel,
         timeout: Duration,
+        use_internal_buffers: bool,
```

Comment:
We should remove this now that this value is passed in the `SolverConfig` right?